presto-main/src/main/java/com/facebook/presto/server/PluginManager.java

```diff
@@ -92,7 +91,6 @@
     @Inject
     public PluginManager(
             NodeInfo nodeInfo,
-            HttpServerInfo httpServerInfo,
```

Comment:
Can you add this change as a separate commit before this one? Because, the other changes are for the test code while this is not.